lldb/test/API/commands/expression/import-std-module/array/Makefile

```diff
@@ -1,3 +1,8 @@
 USE_LIBCPP := 1
 CXX_SOURCES := main.cpp
+
+ifneq ($(OS),Darwin)
+	LD_EXTRAS := -Xlinker --whole-archive -Xlinker --allow-multiple-definition
```

Comment:
Can you elaborate on why this is necessary? Example test failures would be great